src/features/areas/l10n/messageIds.ts

```diff
@@ -13,6 +13,11 @@ export default makeMessages('feat.areas', {
         title: m('Area tags'),
       },
     },
+    controlLabels: {
+      filters: m('Filters'),
+      layers: m('Layers'),
```

Comment:
The issue was not updated to reflect this, but shortly after it was made we discussed changing the name of this section from "Layers" to "Data visualization", so if that is updated here it will be in sync with other work being done :)

Reply:
Okidoki. I've changed it in the translation but left the key as `layers` to be consistent with the rest of the code. Let me know if you would like to see this changed as well.